ChangeLog

```diff
@@ -42,7 +42,14 @@ Version 8.0.0 - DEV
    to convert it to the new style (see 'boolfind' example) or change it to
    include the old run_wrapper.
  - Replace jscolor by coloris.
-
+ - Created insights page in current work queue
+ - Increase performance by disabling class hotinjection
```

Comment:
Maybe better: increase performance by not dynamically loading classes.

Reply:
I agree, will change it.